python/tests/kernel/test_run_kernel.py

```diff
@@ -364,7 +393,9 @@ def simple_list_int(n: int, t: list[int]) -> list[int]:
         return t
 
     results = cudaq.run(simple_list_int, 2, [-13, 5, 42], shots_count=2)
-    # FIXME: Non-const size of stdvec - ReturnToOutputLog does not create output.
+    #FIXME: Non-const size of stdvec - ReturnToOutputLog does not create output.
```

Comment:
Moreover, the intent of the QIR spec is to _not_ support dynamic/variable length vectors/arrays of output.